scripts/scil_validate_and_correct_eddy_gradients.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Validate and correct gradients from eddy outputs
+With full AP-PA eddy outputs a full bvec bval (2x nb of dirs and bval)
+that doesnt fit with the output dwi (1x nb of dir)
+"""
+
+import argparse
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bvec',
+                   help='In bvec file.')
+    p.add_argument('in_bval',
+                   help='In bval file.')
+    p.add_argument('nb_dirs', type=int,
+			       help='Number of directions per DWI.')
+    p.add_argument('out_bvec',
+                   help='Out bvec file.')
+    p.add_argument('out_bval',
+                   help='Out bval file.')
+    add_overwrite_arg(p)
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.in_bvec, args.in_bval])
+    assert_outputs_exist(parser, args, [args.out_bval, args.out_bvec])
+
+    """
+    IN BVEC
+    """
+    in_bvec = np.genfromtxt(args.in_bvec)
+    in_bvec_split = np.hsplit(in_bvec, int(in_bvec.shape[1]/args.nb_dirs))
+    if len(in_bvec_split)==2:
```

Comment:
If nb_dirs doesn't split the bvec array equally, we have a bug at the line after when averaging